src/components/ui/card.svelte

```diff
@@ -1,16 +1,24 @@
 <script>
   export let id = null;
   export let img = null;
+  export let imgWrap = true;
   export let color = '#FFF';
 </script>
 
 <div class="card" class:with-image={img} {id}>
-  <slot name="image">
-    {#if img}
-      <img src={img} style={'background: ' + color} class="image" alt="" />
-    {/if}
-  </slot>
   {#if img}
+    {#if imgWrap}
+      <div class="image">
+        <slot name="image">
+          <img src={img} style="{'background: ' + color}" class="image" alt="" />
```

Comment:
Image size is not bounded by container
![image](https://user-images.githubusercontent.com/11016151/66868488-0e989100-efa6-11e9-9059-d2cdfc21fded.png)
```suggestion
          <img src={img} style="{'max-width: 100%; background: ' + color}" class="image" alt="" />
```

Reply:
Guess it's because different problem. But with your comment, I found it and fixed